packages/argo-run/src/dev.ts

```diff
@@ -29,10 +29,9 @@ export async function dev(...args: string[]) {
   const publicUrl = namedArgument('publicUrl', args);
   const tunnelStarted = Boolean(publicUrl);
   const filename = 'extension.js';
-  const scriptUrl = new URL(
-    `${PUBLIC_PATH}${filename}`,
-    publicUrl || url,
-  ).toString();
+  const script = new URL(`${PUBLIC_PATH}${filename}`, publicUrl || url);
+  const scriptUrl = script.toString();
```

Comment:
Can we make the `URL` object be called `scriptUrl`, and change what was `scriptUrl` to be `scriptHref` or just do `scriptUrl.toString()`/`scriptUrl.href` when we need it?